packages/block-library/src/post-author/edit.js

```diff
@@ -323,12 +327,15 @@ function PostAuthorEdit( {
 			</BlockControls>
 
 			<div { ...blockProps }>
-				{ showAvatar && authorDetails?.avatar_urls && (
+				{ showAvatar && (
 					<div className="wp-block-post-author__avatar">
 						<img
 							width={ avatarSize }
-							src={ authorDetails.avatar_urls[ avatarSize ] }
-							alt={ authorDetails.name }
+							src={
+								authorDetails?.avatar_urls?.[ avatarSize ] ||
+								defaultAvatar
+							}
+							alt={ authorDetails?.name || __( 'Post Author' ) }
```

Comment:
```suggestion
							alt={
								authorDetails?.name || __( 'Default Avatar' )
							}
```

It's a good idea to make it the same as the default alt text for the avatar block.

Reply:
Good suggestion 👍 I’ve updated it to use the same default alt text as the avatar block, so now both stay consistent.